Cargo.toml

```diff
@@ -11,6 +11,7 @@ license = "Apache-2.0/MIT"
 categories = ["multimedia::audio", "api-bindings"]
 readme = "README.md"
 edition = "2018"
+include = ["README.md", "LICENSE-*", "Cargo.toml", "src/"]
```

Comment:
I've just read up on "include" and "exclude" and I think this row is not needed. Just having the files in the repo should be enough to get them included...I think?

Reply:
For some other package it was not working. Cargo can skip them if not include sometime.